services/gateway/starter/src/main/resources/gateway.conf

```diff
@@ -135,6 +135,40 @@ ditto {
         expire-after-write = ${ditto.gateway.cache.publickeys.expiry}
       }
     }
+
+    statistics {
+
+      // how long to wait for messages from other cluster members
+      ask-timeout = 5s
+
+      // how often to update hot entities count for the public
+      update-interval = 15s
+
+      // minimum pause between computations of statistics details
+      details-expire-after = 3s
```

Comment:
Either reduce to less often or add an environment variable so that this may be configured.